langlib/langlib-test/src/test/resources/test-src/valuelib_fromBalString_test.bal

```diff
@@ -177,9 +177,6 @@ function testTableFromBalString() {
     if (tbl is table<any|error>) {
         tbl.add({ id: 3, name: "Jane", grade: 13 });
     }
```

Comment:
Don't remove the test. Update/comment the issue and raise an issue for fromBalString() pointing here

Reply:
Created https://github.com/ballerina-platform/ballerina-lang/issues/30574